src/resmom/catch_child.c

```diff
@@ -106,6 +106,7 @@ extern int		server_stream;
 extern time_t		time_now;
 extern pbs_list_head	mom_polljobs;
 extern unsigned int	pbs_mom_port;
+extern int secondary_pbs_nodefile;
```

Comment:
I think the name is confusing - secondary_pbs_nodefile can be confused with what we doing in other hook

Reply:
also is this aligned? If so, please align the other variables as well